video/rooms/display-participants-video/display-participants-video.m

```diff
@@ -9,21 +9,21 @@ - (void)room:(TVIRoom *)room participantDidConnect:(TVIParticipant *)participant
 /* In the Participant Delegate, we can respond when the Participant adds a Video
 Track by rendering it on screen:*/
 
-#pragma mark - TVIParticipantDelegate
+#pragma mark - TVIVideoViewDelegate
 
 - (void)participant:(TVIParticipant *)participant addedVideoTrack:(TVIVideoTrack *)videoTrack {
     NSLog(@"Participant %@ added a video track",participant.identity);
-    videoTrack.delegate = self;
-    [videoTrack attach:self.remoteMediaView];
+    
+    self.remoteMediaView = [[TVIVideoView alloc] initWithFrame:self.view.bounds delegate:self];
+    [videoTrack addRenderer:self.remoteMediaView];
```

Comment:
I would prefer using `self.remoteView` or `self.remoteVideoView`. Also, you should add the view to your hierarchy after line `18`.